packages/vite/src/node/optimizer/index.ts

```diff
@@ -673,6 +673,28 @@ export function runOptimizeDeps(
                 browserHash: metadata.browserHash,
               })
             }
+          } else {
+            // workaround Firefox warning by removing blank source map reference
+            // https://github.com/evanw/esbuild/issues/3945
+            const output = meta.outputs[o]
+            // filter by exact bytes
+            if (output.bytes === 93) {
```

Comment:
is this the exact bytes of an empty sourcemap?
```suggestion
            // filter by exact bytes of an empty source map
            if (output.bytes === 93) {
```